### PR TITLE
[Bug] Fix possible null exception when calling websocket stop

### PR DESCRIPTION
**Pull request by chrisw000**
Bug fix for #190 

There is also a change that corrects an issue that can occur in the historic candles area - when pulling very old data it's possible to get into an infinite loop where it keeps pulling the oldest candles over and over.

### CODE REVIEW

File: CoinbasePro/WebSocket/WebSocket.cs [hunk 1]
```diff
@@ -92,6 +92,12 @@ public class WebSocket : IWebSocket
 
         public void Stop()
         {
+            if (webSocketFeed == null)
```
Comment (dougdellolio) [inline CoinbasePro/WebSocket/WebSocket.cs]: hey @chrisw000. just trying to understand the case when this happens. does this only occur if the user is calling stop before start is called?

Comment (chrisw000) [reply CoinbasePro/WebSocket/WebSocket.cs]: Correct. Obviously its possible to handle it outside this codebase - but its cleaner to have it track its own state.

Comment (dougdellolio) [reply CoinbasePro/WebSocket/WebSocket.cs]: okay cool, sure. would you mind making the log message a little more detailed? maybe something along the lines of like `Websocket did not stop successfully as the feed has not been started yet." also feels like it should be a warning as the user is doing something incorrectly.... then lets merge it

Comment (chrisw000) [reply CoinbasePro/WebSocket/WebSocket.cs]: OK, changed message to:
Websocket did not attempt to stop as the feed has not been started yet

and changed it to a warning.

Yeah - I get this happen if something else odd external has happened, so the app is shutting down before its got going. Things like network issue / new install has wrong configuration / bad db connection string etc.

**Pull request merged**